node-tests/blueprints/acceptance-test-test.js

```diff
@@ -16,9 +16,9 @@ describe('Acceptance: ember generate and destroy acceptance-test', function() {
     return emberNew()
       .then(() => emberGenerateDestroy(args, (file) => {
         expect(file('tests/acceptance/foo-test.coffee'))
-          .to.contain("`import Ember from 'ember'`")
-          .to.contain("`import { module, test } from 'qunit'`")
-          .to.contain("`import startApp from 'my-app/tests/helpers/start-app'`")
+          .to.contain("import Ember from 'ember")
```

Comment:
It's just checking that the file contains that string, and `'ember'` does contain `'ember`. I think it's fine.